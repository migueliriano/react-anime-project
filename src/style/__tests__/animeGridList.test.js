import React from 'react';

import GridContainer from '../animeGridList';

describe('animeGridList style components', () => {
  it('Should render a GridContainer Component', () => {
    const tree = renderer.create(<GridContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
