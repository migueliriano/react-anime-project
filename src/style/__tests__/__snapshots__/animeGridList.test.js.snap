// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`animeGridList style components Should render a GridContainer Component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  padding: 30px;
  max-width: 1600px;
  margin: 0 auto;
}

<section
  className="c0"
/>
`;
